### PR TITLE
Add Plone 5.1 support and drop archetypes support.

### PR DESCRIPTION
**Pull request by tinagerber**
closes #https://github.com/4teamwork/izug.organisation/issues/1454

### CODE REVIEW

File: ftw/tagging/profiles/default_plone5/portlets.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<portlets>
+
+    <portlet
```
Comment (busykoala) [inline ftw/tagging/profiles/default_plone5/portlets.xml]: It's xml, but I'd still change the indents...

File: setup.py [hunk 2]
```diff
@@ -56,9 +56,6 @@
 
         # Plone
         'Plone',
-        'archetypes.schemaextender',
-        'Products.AddRemoveWidget',
-        'Products.Archetypes',
         'Products.CMFCore',
         'Products.CMFPlone',
         'ftw.keywordwidget',
```
Comment (busykoala) [inline setup.py]: @maethu, do we require `ftw.keywordwidget >= 2.0`.

File: README.rst [hunk 3]
```diff
@@ -49,15 +41,15 @@ Enable tagging on your DX types with a behavior:
...
+Optionally define tag roots:
 
 ::
 
   <class class="path.to.my.class">
     <implements interface="ftw.tagging.interfaces.tagging.ITagRoot" />
   </class>
 
-For DX, you could alternatively use a behavior to define tag roots:
+You could alternatively use a behavior to define tag roots:
```
Comment (djowett-ftw) [inline README.rst]: I think the behavior should be the primary way (or perhaps only way) of specifying a tag root :-)

File: ftw/tagging/profiles/uninstall_plone5/metadata.xml [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (djowett-ftw) [inline ftw/tagging/profiles/uninstall_plone5/metadata.xml]: I don't think an empty metadata.xml is necessary in an uninstall profile at least (I didn't put one in ftw.globalstatusmessage anyway)

File: ftw/tagging/profiles/default_plone5/skins.xml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<object name="portal_skins" allow_any="False" cookie_persistence="False">
+
+    <object name="ftw.tagging"
+            meta_type="Filesystem Directory View"
+            directory="ftw.tagging:skins/ftw_tagging" />
+
+    <skin-path name="*">
+        <layer name="ftw.tagging"
+               insert-after="custom" />
```
Comment (djowett-ftw) [inline ftw/tagging/profiles/default_plone5/skins.xml]: Shame we can't lose skins ;-)

but `insert-after` is wrong - you mean `insert-before`..... which means `custom` can still override through the web (though we'd prob never do that in practise)

**Comment by tinagerber** [comment]
Ok, I made the desired changes.
I also z3cformified the portlet. 😄

**Comment by busykoala** [comment]
> LGTM too - but we just want to clarify the version of keywordwidget as per @busykoala's [comment](https://github.com/4teamwork/ftw.tagging/pull/9/files#r333961619)

I think we should not pin it there since it would only be for plone 5. If we did it for plone 5 only (in an extra) there is no use (like helping the user installing the package understand that it should be pinned differently). We could consider putting this in one of our test.cfgs though.

**Comment by djowett-ftw** [comment]
> I think we should not pin it there since it would only be for plone 5. If we did it for plone 5 only (in an extra) there is no use (like helping the user installing the package understand that it should be pinned differently). We could consider putting this in one of our test.cfgs though.

So you're suggesting a change to README.md? Sounds fine to me

**Comment by tinagerber** [comment]
I have now also added an uninstall test.

**Pull request merged**